bqetl_project.yaml

```diff
@@ -215,3 +215,87 @@ dry_run:
+  - sql/moz-fx-data-shared-prod/telemetry_derived/clients_daily_keyed_scalar_aggregates_v1/query.sql
+  - sql/moz-fx-data-shared-prod/telemetry_derived/clients_daily_scalar_aggregates_v1/init.sql
+  - sql/moz-fx-data-shared-prod/telemetry_derived/clients_daily_scalar_aggregates_v1/query.sql
+  - sql/moz-fx-data-shared-prod/telemetry_derived/clients_histogram_aggregates_v1/init.sql
+  - sql/moz-fx-data-shared-prod/telemetry_derived/clients_histogram_aggregates_v1/query.sql
+  - sql/moz-fx-data-shared-prod/telemetry_derived/clients_histogram_probe_counts_v1/query.sql
+  - sql/moz-fx-data-shared-prod/telemetry_derived/clients_scalar_probe_counts_v1/query.sql
+  - sql/moz-fx-data-shared-prod/telemetry_derived/core_clients_daily_v1/query.sql
+  - sql/moz-fx-data-shared-prod/telemetry_derived/core_clients_last_seen_v1/init.sql
+  - sql/moz-fx-data-shared-prod/telemetry_derived/core_clients_last_seen_v1/query.sql
+  - sql/moz-fx-data-shared-prod/telemetry_derived/core_live/view.sql
+  - sql/moz-fx-data-shared-prod/telemetry_derived/devtools_events_amplitude_v1/view.sql
+  - sql/moz-fx-data-shared-prod/telemetry_derived/error_aggregates/query.sql
+  - sql/moz-fx-data-shared-prod/telemetry_derived/event_events_v1/init.sql
+  - sql/moz-fx-data-shared-prod/telemetry_derived/event_events_v1/query.sql
+  - sql/moz-fx-data-shared-prod/telemetry_derived/events_live/view.sql
+  - sql/moz-fx-data-shared-prod/telemetry_derived/experiment_enrollment_aggregates_v1/init.sql
+  - sql/moz-fx-data-shared-prod/telemetry_derived/experiment_enrollment_aggregates_v1/query.sql
+  - sql/moz-fx-data-shared-prod/telemetry_derived/experiments_daily_active_clients_v1/init.sql
+  - sql/moz-fx-data-shared-prod/telemetry_derived/firefox_desktop_exact_mau28_by_client_count_dimensions_v1/query.sql
+  - sql/moz-fx-data-shared-prod/telemetry_derived/fxa_users_services_daily_v1/query.sql
+  - sql/moz-fx-data-shared-prod/telemetry_derived/fxa_users_services_last_seen_v1/query.sql
+  - sql/moz-fx-data-shared-prod/telemetry_derived/glam_user_counts_v1/query.sql
+  - sql/moz-fx-data-shared-prod/telemetry_derived/latest_versions/query.sql
+  - sql/moz-fx-data-shared-prod/telemetry_derived/main_events_v1/init.sql
+  - sql/moz-fx-data-shared-prod/telemetry_derived/main_events_v1/query.sql
+  - sql/moz-fx-data-shared-prod/telemetry_derived/scalar_percentiles_v1/query.sql
+  - sql/moz-fx-data-shared-prod/telemetry_derived/smoot_usage_desktop_v2/query.sql
+  - sql/moz-fx-data-shared-prod/telemetry_derived/smoot_usage_fxa_v2/query.sql
+  - sql/moz-fx-data-shared-prod/telemetry_derived/smoot_usage_new_profiles_v2/query.sql
+  - sql/moz-fx-data-shared-prod/udf_js/jackknife_percentile_ci/udf.sql
+  - sql/moz-fx-data-shared-prod/udf_js/bootstrap_percentile_ci/udf.sql
+  - sql/moz-fx-data-shared-prod/udf_legacy/contains.sql
+  - sql/moz-fx-data-shared-prod/udf_legacy/date_format.sql
+  - sql/moz-fx-data-shared-prod/udf_legacy/date_trunc.sql
+  - sql/moz-fx-data-shared-prod/udf_legacy/to_iso8601.sql
```

Comment:
I think we should run formatting on most of these in a follow-up PR so that we don't have to maintain this list here